src/common/components/nav/nav.bundle.js

```diff
@@ -7,11 +7,11 @@
   // angular.js to load itself into
   const angular = (window.angular = {});
 
-  require('./nav.component');
+  const nav = require('./nav.component');
```

Comment:
Does this work?  I believe the nav.component source needs to be put into the bundle.